README.md

```diff
@@ -120,71 +168,100 @@ from torchvision import datasets, transforms
+Whether you want to reach the same quality faster or get better quality within your current budget, Explorer can help you decide which speedup methods to use.
+We update this data regularly as we add new methods and develop better recipes.
+
+<p align="center">
+<img src="https://storage.googleapis.com/docs.mosaicml.com/images/methods/explorer.png"/>
+</p>
+
+As an example, here are two performant recipes, one for ResNet-101 on ImageNet, and the other for GPT-2 on OpenWebText, on 8xA100s: 
+
+#### ResNet-101
+
+Name|Functional|tl;dr|Benchmark|Speed Up
+----|----------|-----|---------|--------
+[Blur Pool](https://github.com/mosaicml/composer/tree/dev/composer/algorithms/blurpool)|`cf.apply_blurpool`|[Applies an anti-aliasing filter before every downsampling operation.](https://arxiv.org/abs/1904.11486)|ResNet-101|1.2x
+[Channels Last](https://github.com/mosaicml/composer/tree/dev/composer/algorithms/channels_last)|`cf.apply_`<br>`channels_last`|[Uses channels last memory format (NHWC).](https://pytorch.org/tutorials/intermediate/memory_format_tutorial.html)|ResNet-101|1.5x
+[Label Smoothing](https://github.com/mosaicml/composer/tree/dev/composer/algorithms/label_smoothing)|`cf.smooth_labels`|[Smooths the labels with a uniform prior.](https://arxiv.org/abs/1512.00567)|ResNet-101|1.5x
+[MixUp](https://github.com/mosaicml/composer/tree/dev/composer/algorithms/mixup)|`CF.mixup_batch`|[Blends pairs of examples and labels.](https://arxiv.org/abs/1710.09412)|ResNet-101|1.5x
+[Progressive Resizing](https://github.com/mosaicml/composer/tree/dev/composer/algorithms/progressive_resizing)|`cf.resize_batch`|[Increases the input image size during training.](https://github.com/fastai/fastbook/blob/780b76bef3127ce5b64f8230fce60e915a7e0735/07_sizing_and_tta.ipynb)|ResNet-101|1.3x
+[SAM](https://github.com/mosaicml/composer/tree/dev/composer/algorithms/sam)|`N/A`|[SAM optimizer measures sharpness of optimization space.](https://arxiv.org/abs/2010.01412)|ResNet-101|1.5x
+**Composition** | `N/A` | **Cheapest: $49 @ 78.1% Acc** | **ResNet-101** | **3.5x**
+
+#### GPT-2
+
+Name|Functional|tl;dr|Benchmark|Speed Up
+----|----------|-----|---------|--------
+[Alibi](https://github.com/mosaicml/composer/tree/dev/composer/algorithms/alibi)|`cf.apply_alibi`|[Replace attention with AliBi](https://arxiv.org/abs/2108.12409v1)|GPT-2|1.6x
+[Seq Length Warmup](https://github.com/mosaicml/composer/tree/dev/composer/algorithms/seq_length_warmup)|`cf.set_batch_`<br>`sequence_length`|[Progressively increase sequence length.](https://arxiv.org/abs/2108.06084)|GPT-2|1.5x
+**Composition** | `N/A` | **Cheapest: $150 @ 24.11 PPL** | **GPT-2** | **1.7x**
+
+## ⚙️ What benchmarks does Composer support?
+
+Composer uses a _benchmark_ as a term to denote a particular model trained on a particular dataset in a standardized, reproducible way. A benchmark is a specific model trained for a task, where a task is defined as a dataset with a specific loss function.
```

Comment:
This is a bit verbose.

```suggestion
A _benchmark_ is a particular model trained on a particular dataset with a specific loss function.
```

I think it is implied that a benchmark is reproducible?

Also, is scheduling part of a benchmark?